composites/Plugin/SnippetEditor/serialization.js

```diff
@@ -64,128 +66,240 @@ export function serializeEditor( rawContent ) {
 }
 
 /**
- * Unserializes a tag into a string.
+ * Determines the title for a given name.
  *
- * @param {string} serializedTag The serialized tag.
+ * @param {Array} replacementVariables All the available replacment variables.
+ * @param {string} name The name to find the title for.
```

Comment:
Sorry to be a bit picky, but do we align variables and description or not? 🙂 See line 250.